qa-nde.planx-pla.net/manifest.json

```diff
@@ -31,7 +31,7 @@
     "environment": "qaplanetv1",
     "hostname": "qa-nde.planx-pla.net",
     "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/ndhdictionary/3.5.3/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/ndhdictionary/3.5.4/schema.json",
```

Comment:
i'm unsure about this one, NDE prod is using 3.1.14. i think the best thing to do would be to remove the etl mapping like in prod https://github.com/uc-cdis/cdis-manifest/tree/master/niaiddata.org
and leave the DD as is

Reply:
i don't think the NDE portal actually cares about DD or ETL mapping anyways, they are not being used

Reply:
OK makes sense!